packages/react-provider/src/Provider.tsx

```diff
@@ -0,0 +1,71 @@
+import { useMergedRefs } from '@fluentui/react-hooks';
+import { getSlots, makeMergeProps } from '@fluentui/react-utils';
+import * as React from 'react';
+
+import { ThemeContext, ProviderContextValue, useFluent } from './context';
+import { PartialTheme } from '@fluentui/react-theme';
+import { ThemeProviderState, useThemeProviderState } from './ThemeProvider';
+
+export interface ProviderProps extends Partial<ProviderContextValue> {
```

Comment:
can we have `props` not extending anything else? It took me some time to realise `theme` is not the only prop.

Reply:
Done 👍